mitmproxy/tools/console/flowview.py

```diff
@@ -115,26 +115,21 @@ def view_tcp_stream(self) -> urwid.Widget:
         if not flow.messages:
             return searchable.Searchable([urwid.Text(("highlight", "No messages."))])
 
-        from_client = None
-        messages = []
-        for message in flow.messages:
-            if message.from_client is not from_client:
-                messages.append(message.content)
-                from_client = message.from_client
-            else:
-                messages[-1] += message.content
```

Comment:
The way we store TCP flow contents as "messages" is a bit disconnected from reality. In practice, TCP is just a bidirectional stream with no defined messages boundaries. A "message" in mitmproxy is essentially just "stuff we happened to read from the socket in a single syscall". Given this arbitrariness, I think it's correct to keep the code here which concatenates adjacent messages sent by the same side. Any objections?

Reply:
At first I just didn't got the purpose of this lines after some time struggling to understand them I've tried to remove them and it worked fine. So I've came to conclusion that we can remove them 😄 

Your explanation definitely makes sense, no objections! I'll resurrect this code!

Reply:
However I've decided to rewrite this part in a way that I believe is a little bit more simple for unfamiliar code reader 😃